README.md

```diff
@@ -381,12 +381,106 @@ rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.
 
 ### Writing Release Notes
 
-Release notes are maintained in the [change log](CHANGELOG.md) and follow guidelines
-based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). We include additional
-guidelines for increased readability and consistency:
-
-* Changes should be grouped by section, either as an enhancement, bug fix, or
-  internal change
-* References to code or configuration should render as literals
-* Bug fixes should link to bug reports (GitHub Issues or Jira items)
-* Features or enhancements should link to formal RFEs (GitHub Issues or Jira items)
+Release notes are maintained in the [changelog](CHANGELOG.md) and follow
+guidelines based on [keep a changelog](https://keepachangelog.com/en/1.0.0/).
+This section describes additional guidelines, conventions, and the overall
+process for writing release notes.
+
+#### Guidelines
+
+* Each release should contain release notes
+* Changes should be applicable to at least one of the six types listed below
+* Use literals for code and configuration
+  - ``defaultScanSettingsSchedule``
+  - ``nodeSelector``
+* Write your notes with users as the audience
+* Keep notes as short as possible and link to additional documentation when necessary
+  - Bug fixes should link to bug reports (GitHub Issues or Jira items)
+  - Features or enhancements should link to formal RFEs (GitHub Issues or Jira
+    items)
```

Comment:
Do we want to suggest a writing style? I noticed that most of the README is written in active voice. Do we want to add that to this list? Or are there other conventions that we should be documenting here?

Reply:
Umm whatever some other project does? Both work for me

Reply:
I like active voice + imperative. But I often drift (read too much Phillip K. Dick as a kid?) . So it would be good to have a reminder and example.
I like the advice in https://wiki.openstack.org/wiki/GitCommitMessages#Information_in_commit_messages

Reply:
Sounds good, do we want to include the commit message guidelines here or in a separate change?

Reply:
sgtm.